docs/en/jump-start-extdev-plasma-us1.md

```diff
@@ -48,6 +48,11 @@ cd ../loom-jump-start
 
 ## Start the node
 
+> Before starting the node, check the open files limit on your system. See the [Open Files Limit](#open-files-limit) section for more details
```

Comment:
-> `> NOTE: Check the open files limit on your system before starting the node! See [Open Files Limit](#open-files-limit)`

Reply:
> -> `> NOTE: Check the open files limit on your system before starting the node! See [Open Files Limit](#open-files-limit)`

Updated.